src/site/content/en/blog/serial/index.md

```diff
@@ -393,15 +393,21 @@ access a serial port, it should monitor the `connect` and `disconnect` events.
 
 ```js
 navigator.serial.addEventListener("connect", (event) => {
-  // TODO: Automatically open event.port or warn user a port is available.
+  // TODO: Automatically open event.target or warn user a port is available.
 });
 
 navigator.serial.addEventListener("disconnect", (event) => {
-  // TODO: Remove |event.port| from the UI.
+  // TODO: Remove |event.target| from the UI.
   // If the serial port was opened, a stream error would be observed as well.
 });
 ```
 
+{% Aside %}
+Prior to Chrome 89 the `connect` and `disconnect` events fired a custom
+`SerialConnectionEvent` object with the affected `SerialPort` interface
+available as the `port` attribute.
```

Comment:
You may want to recommend `event.port || event.target` as a way to handle the transition to the new attribute.

Reply:
Done. Thanks!